.github/ISSUE_TEMPLATE/portfolio-review.yml

```diff
@@ -13,6 +13,13 @@ body:
       placeholder: "e.g.: https://<your-username>.github.io/portfolio"
     validations:
       required: true
+  - type: input
+    id: link
+    attributes:
+      label: GitHub Repo Link
+      placeholder: "e.g.: https://github.com/YOUR_USERNAME/REPO"
+    validations:
+      required: true
```

Comment:
Do we need to make this required? This is for the portfolio review. 🤔

Reply:
I think required is important 👍  I only want to review open source portfolios